.circleci/config.yml

```diff
@@ -306,10 +306,22 @@ jobs:
       - save_cache: *save-jest-cache
       - store-artifacts-and-test-results
 
+  danger:
+    <<: *defaults
+    steps:
+      - checkout
```

Comment:
This was really slow because we lost the git cache 🙄 

Let's make this:

```yaml
      - restore_cache: *restore-git-cache
      - checkout
      - run: #…
```